src/main/java/ru/testtask/model/Project.java

```diff
@@ -17,6 +17,7 @@
 
     @Id
     @Getter
+    @Setter
```

Comment:
All fields has getters and setters. You can remove all annotations `@Getter` and `@Setter` and use `@Data` instead